bin/vat doesn't work

There is an error when running the encouragementBot from the README:
```
michael$ bin/vat run demo/encouragementBot
/Users/michael/src/SwingSet/bin/vat:1
Error: Cannot find module '../src/devices'
Require stack:
- /Users/michael/src/SwingSet/bin/vat
    at Object.<anonymous> (/Users/michael/src/SwingSet/bin/vat:10:26)
    at Module._compile (internal/modules/cjs/loader.js:816:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:827:10)
    at Module.load (internal/modules/cjs/loader.js:685:32)
    at Function.Module._load (internal/modules/cjs/loader.js:620:12)
michael$
```

--- a/src/devices/index.js
+++ b/src/devices/index.js
@@ -1,0 +1,5 @@
+// Export all the default exports of the driver top-halves.
+export { default as buildChannel } from './channel';
+export { default as buildInbound } from './inbound';
+export { default as buildOutbox } from './outbox';
+export { default as buildSharedStringTable } from './sharedTable';

